data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1217 @@
+			`	"You're with Etarip, right?"`
+			choice
+				`	"That's correct - you're heading down south?"`
+			`	The man rubs his hands together and smiles. "Yeah, yeah! That's right. Big things are happening. So, can we go?"`
+			choice
+				`	"I might need to take care of a few things first, but I'll show you to your-"`
+			`	The man vigorously shakes your hand before you finish. "Great, great!" He walks past you and ducks into your ship.`
+				
+	on complete
+		event "Etarip Delivery 4" 4 12
+		conversation
+			`As you land on Deep, you take note of the eerily dark ocean surrounding the main island. You find the businessman lounging in his room in a brightly-colored floral shirt, all traces of anxiety gone.`
+			choice
+				`	"We're here - I'll show you out."`
+			`	The man stands, stretches, and shakes your hand gently. "You're a miracle worker, captain. Thanks for the help." You show him to the exit and wave him off.`
+		
+event "Defer Etarip Delivery 3"
+#* * *  (Mission 3 END)  * * *
+
+
+
+#* * * * *  (Mission 4)  * * * * *
+event "Etarip Delivery 4"
+
+mission "Etarip Delivery [4]"
+	name "Etarip Delivery to <planet>"
+	description "Deliver <cargo> to <destination> for Etarip Enterprises."
+	blocked `Alexandra contacts you just as you touch down on <origin>. "We have a job for you, <first>, but you're going to need <capacity>. Remember - we have a contract! Drop whatever you're doing and come back."`
+	landing
+	cargo "Personal Defense Weapons" 10
+	deadline 2
+	infiltrating
+	stopover
+		government Republic
+		outfits "Laser Rifle"
+		distance 3 5
+	
+	to offer
+		or
+			and
+				has "Etarip Delivery [3]: done"
+				has "event: Etarip Delivery 4"
+				not "deferred etarip delivery 4"
+			and
+				has "event: Defer Etarip Delivery 4"
+	source
+		government Republic
+	destination
+		government Republic
+		attributes "near earth"
+		attributes "urban"
+	
+	on offer
+		conversation
+			branch deferred
+				"deferred etarip delivery 4" == 1
+			`"Good afternoon, <first>!" Alexandra's cheerful voice fills the cabin as you perform final landing checks.`
+			choice
+				`	"Hey, Alexandra. What have you got for me?"`
+					goto business
+				`	"That's funny - I didn't think angels used comm lines."`
+					goto funny
+				`	"I take it this isn't a pleasure call?"`
+			`	"Not quite, but I guess that all depends on your attitude!"`
+				goto business
+			label funny
+			`	You hear Alexandra sigh. "A little advice, <first>? You're trying way too hard. But this angel has more important things to work on."`
+			label business
+			`	"There's a private security firm on <destination> that needs a batch of personal defense weapons. Pick them up at the outfitter on <stopovers>, and make sure they get to <planet> by <day>."`
+			label questions
+			choice
+				`	"I'm on it."`
+					goto yes
+				`	"Personal defense weapons?"`
+				`	"Isn't transporting weapons illegal?"`

```

Comment:
I'm not sure why your captain would ask this. Given that weapons are legal to purchase and own in the Endless Sky world, they obviously need to be transported to the outfitter by *someone.* Also "dueling sabers" are already listed among the commodities lists so clearly some weapons are legal to transport.

Maybe change the phrasing to something like "Isn't it illegal for private couriers to transport weapons?" or "It's illegal to transport anything more dangerous than class III weapons; what class do these fall under?"